src/pages/Dashboard/Actions/AddActionCard.vue

```diff
@@ -34,11 +35,29 @@ export default {
         select: {
           title: '...'
         }
-      }
+      },
+      SLATypes: [
+        { name: 'starts but does not finish', enum: 'STARTED_NOT_FINISHED' },
```

Comment:
Note that the check duration begins at the flow 'Running' state transition time.

Reply:
Might be helpful in some sort of tooltip?